modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -646,6 +686,24 @@ export default class AvgRTPStatsReporter {
                     frameRate, true /* local */, VideoType.DESKTOP));
         }
 
+        if (resolution) {
+            this._avgRemoteCameraPixels.addNext(
```

Comment:
what is the scope of this...is it always tracking whichever camera stream is on the main stage?